buildbot/master.cfg

```diff
@@ -821,6 +863,8 @@ BuildmasterConfig = {
                       factory=make_dolphin_debian_build("wip")),
         BuilderConfig(name="wip-android", slavenames=["ubuntu"],
                       factory=make_android_package("wip")),
+        BuilderConfig(name="wip-freebsd", slavenames=["freebsd"],
```

Comment:
Release build will check on every master commit. IMO it's worth to test it as merging may generate issues.